easybuild/easyconfigs/p/PaStiX/PaStiX-5.2.3-foss-2017b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'PaStiX'
+version = '5.2.3'
+
+homepage = 'http://pastix.gforge.inria.fr/'
+description = """PaStiX (Parallel Sparse matriX package) is a scientific library that provides a high performance
+ parallel solver for very large sparse linear systems based on direct methods."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://gforge.inria.fr/frs/download.php/file/36212/']
+sources = ['%(namelower)s_%(version)s.tar.bz2']
+checksums = ['641978e6b0607e201a409549adc78d5618ec159f44cafc7bddeaca4f4bee1e47']
+
+dependencies = [
+    ('hwloc', '1.11.7'),
+    ('SCOTCH', '6.0.4'),
+]
+
+skipsteps = ['configure']
+
+start_dir = 'src'
+
+prebuildopts = "cp -a config/LINUX-GNU.in config.in && "
+buildopts = 'BLASLIB="-L$BLAS_LIB_DIR $LIBBLAS" HWLOC_HOME=$EBROOTHWLOC SCOTCH_HOME=$EBROOTSCOTCH '
+# take control over compiler flags
+buildopts += 'CCFOPT="$CFLAGS" '
+# fix value for $MPCXXPROG, default uses 'mpic++' which is incorrect
+buildopts += 'MPCXXPROG="$MPICXX -Wall" '
```

Comment:
@smoors In the existing `PaStiX`, we use the following to ensure that the right compiler is being used (by the MPI compiler wrapper).

```python
buildopts += 'MPCXXPROG="$MPICXX -cxx=$CXX -Wall" '
```

Is there are reason you changed this? I think it makes sense to keep it like that, to avoid accidentally build with `gcc` when the `intel` toolchain is used?

Reply:
@boegel I tried it first like this, but it failed during the test step:
g++: error: unrecognized command line option ‘-cxx=g++’
It is already using g++, no need to add it :)

Reply:
OK, the `-cxx=` is probably specific to the Intel MPI wrappers then, fine...